<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerProjectiveTextureMappingModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerProjectiveTextureMappingModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="InputsCollapsibleButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Inputs</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="collapsed">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="DisplayLabel">
        <property name="text">
         <string>Display:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="DisplayNodeComboBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLProjectiveTextureMappingDisplayNode</string>
         </stringlist>
        </property>
        <property name="showHidden">
         <bool>true</bool>
        </property>
        <property name="baseName">
         <string>ProjectiveTextureMapping</string>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
        <property name="addEnabled">
         <bool>true</bool>
        </property>
        <property name="renameEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="ViewLabel">
        <property name="text">
         <string>View:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="qMRMLDisplayNodeViewComboBox" name="ViewCheckableNodeComboBox">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLViewNode</string>
         </stringlist>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="DisplayCollapsibleButton">
     <property name="text">
      <string>Display</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="TextureLabel">
        <property name="text">
         <string>Texture:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="qMRMLNodeComboBox" name="TextureNodeComboBox">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLScalarVolumeNode</string>
          <string>vtkMRMLVectorVolumeNode</string>
         </stringlist>
        </property>
        <property name="hideChildNodeTypes">
         <stringlist/>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="renameEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="CameraLabel">
        <property name="text">
         <string>Camera:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="qMRMLNodeComboBox" name="CameraNodeComboBox">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLCameraNode</string>
         </stringlist>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="renameEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="SurfaceModelNodeComboBox">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLModelNode</string>
         </stringlist>
        </property>
        <property name="hideChildNodeTypes">
         <stringlist/>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="renameEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="SurfaceModelLabel">
        <property name="text">
         <string>Surface Model:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="ctkCheckBox" name="VisibilityCheckBox">
        <property name="text">
         <string/>
        </property>
        <property name="indicatorIcon">
         <iconset resource="../../../../../../../Programme/Slicer/Libs/MRML/Widgets/Resources/qMRMLWidgets.qrc">
          <normaloff>:/Icons/VisibleOn.png</normaloff>
          <normalon>:/Icons/VisibleOff.png</normalon>:/Icons/VisibleOn.png</iconset>
        </property>
        <property name="indicatorIconSize">
         <size>
          <width>21</width>
          <height>21</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="VisibilityLabel">
        <property name="text">
         <string>Visibility:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="ctkSliderWidget" name="ZToleranceSliderWidget">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="pageStep">
         <double>0.100000000000000</double>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="value">
         <double>0.050000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="ZToleranceLabel">
        <property name="text">
         <string>Z Tolerance:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="ColorMapLabel">
        <property name="text">
         <string>Color Map:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="qMRMLColorTableComboBox" name="ColorMapNodeComboBox">
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
        <property name="selectNodeUponCreation">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLCheckableNodeComboBox</class>
   <extends>qMRMLNodeComboBox</extends>
   <header>qMRMLCheckableNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLColorTableComboBox</class>
   <extends>qMRMLNodeComboBox</extends>
   <header>qMRMLColorTableComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLDisplayNodeViewComboBox</class>
   <extends>qMRMLCheckableNodeComboBox</extends>
   <header>qMRMLDisplayNodeViewComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCheckBox</class>
   <extends>QCheckBox</extends>
   <header>ctkCheckBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../../../../Programme/Slicer/Libs/MRML/Widgets/Resources/qMRMLWidgets.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>qSlicerProjectiveTextureMappingModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>CameraNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerProjectiveTextureMappingModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>SurfaceModelNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerProjectiveTextureMappingModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ViewCheckableNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerProjectiveTextureMappingModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>TextureNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DisplayNodeComboBox</sender>
   <signal>currentNodeChanged(bool)</signal>
   <receiver>DisplayCollapsibleButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DisplayNodeComboBox</sender>
   <signal>currentNodeChanged(vtkMRMLNode*)</signal>
   <receiver>ViewCheckableNodeComboBox</receiver>
   <slot>setMRMLDisplayNode(vtkMRMLNode*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerProjectiveTextureMappingModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>DisplayNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerProjectiveTextureMappingModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ColorMapNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
